### PR TITLE
feat(pkger): adds histogram chart support

### PR DESCRIPTION
**Pull request by dearyhud**
Closes #

Describe your proposed changes here.

### CODE REVIEW

File: pkger/models.go [hunk 1]
```diff
@@ -887,10 +906,12 @@ func (c chart) validProperties() []ValidationErr {
 	switch c.Kind {
 	case chartKindGauge:
 		fails = append(fails, c.Colors.hasTypes(colorTypeMin, colorTypeThreshold, colorTypeMax)...)
-	case chartKindScatter:
-		fails = append(fails, c.Axes.hasAxes("x", "y")...)
 	case chartKindHeatMap:
 		fails = append(fails, c.Axes.hasAxes("x", "y")...)
+	case chartKindHistogram:
+		fails = append(fails, c.Axes.hasAxes("x")...)
```
Comment (dearyhud) [inline pkger/models.go]: acc'd the UI and the histogram datamodel, histogram charts only requires an x-axis. I could be mistaken

File: pkger/clone_resource.go [hunk 2]
```diff
@@ -155,6 +167,14 @@ func convertChartToResource(ch chart) Resource {
 		r[fieldChartBinSize] = ch.BinSize
 	}
 
+	if ch.BinCount != 0 {
+		r[fieldChartBinCount] = ch.BinCount
+	}
+
+	if ch.Position != "" {
+		r[fieldChartPosition] = ch.Position
```
Comment (jsteenb2) [inline pkger/clone_resource.go]: you can add this to the slice below that starts on line 188, ig ignores empty strings and doesn't provide the field 👍

File: pkger/clone_resource.go [hunk 3]
```diff
@@ -155,6 +167,14 @@ func convertChartToResource(ch chart) Resource {
 		r[fieldChartBinSize] = ch.BinSize
 	}
 
+	if ch.BinCount != 0 {
+		r[fieldChartBinCount] = ch.BinCount
```
Comment (jsteenb2) [inline pkger/clone_resource.go]: you can add this to the slice that starts on line 203

File: pkger/models.go [hunk 4]
```diff
@@ -733,6 +734,8 @@ const (
 	fieldChartYPos          = "yPos"
 	fieldChartBinSize       = "binSize"
 	fieldChartDomain        = "domain"
+	fieldChartBinCount      = "binCount"
```
Comment (jsteenb2) [inline pkger/models.go]: nit: can you add these in alphabetical order, looks like last 3 didn't get in order

**Pull request merged**